bodhi/tests/server/services/test_releases.py

```diff
@@ -41,6 +41,16 @@ def setUp(self):
             branch=u'f22')
 
         self.db.add(release)
+
+        packages = [
+            Package(name=u'python', type=ContentType.rpm),
+            Package(name=u'json', type=ContentType.rpm),
+            Package(name=u'nodejs', type=ContentType.rpm),
+            Package(name=u'git', type=ContentType.module)
+        ]
+        for pkg in packages:
+            self.db.add(pkg)
```

Comment:
Let's make sure there's a test for packages that already exist, and another test for packages that do not already exist. The API should create packages that do not exist automatically.

Reply:
That will ignore invalid package names due to typing error. 

How about a flag such as ```--create```, which if passed, creates packages that don't exist. Otherwise, prevents invalid package names.

Reply:
To solve this problem, we can use the koji client to verify that packages we haven't seen before exist in Koji. This way we can handle new packages while also verifying that they really exist.